src/sentry/receivers/sentry_apps.py

```diff
@@ -75,6 +86,18 @@ def send_comment_webhooks(organization, issue, user, event, data=None):
             user_id=(user.id if user else None),
             data=data,
         )
+    if features.has("organizations:sentry-functions", organization, actor=user):
+        if user:
+            try:
+                data["user"] = serialize(User.objects.get(id=user.id), user, UserSerializer())
+            except User.DoesNotExist:
+                pass
+        try:
+            data["comment"] = serialize(Group.objects.get(id=issue.id))
```

Comment:
@markstory Do you think it makes sense to serialize these before passing it into the celery task or maybe we should just query for it inside the task so we don't have to serialize it at all?

Reply:
I made the decision to do it outside the celery task because it would be repeated many times, depending on how many sentry functions the organization has, but I'm open to changing it!

Reply:
Your concern definitely makes sense. That being said, it seems unlikely folks would have that many Sentry functions for an organization. A few extra indexed queries probably isn't a big deal. But I'm curious what @markstory says.